src/middlewares/isLogged.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const jwt = require('jsonwebtoken');
+
+const { SECRET_KEY } = process.env;
+
+// verify token cookie middleware
+const isLogged = (req, res, next) => {
+  const { token } = req.cookies;
+
+  if (token) {
+    try {
+      const loggedUser = jwt.verify(token, SECRET_KEY);
+      if (loggedUser) {
+        req.user = loggedUser;
+        next();
+      }
+    } catch (error) {
+      res
+        .clearCookie('token')
+        .status(400)
+        .json({ error: 'token expired please login' });
+    }
+  }
+
+  res
+    .clearCookie('token')
+    .status(400)
```

Comment:
status code 401 because the user is not unauthorized to see the content or do some actions